homeassistant/components/tplink/__init__.py

```diff
@@ -76,6 +77,10 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up TPLink from a config entry."""
     config_data = hass.data[DOMAIN].get(ATTR_CONFIG)
 
+    device_registry = dr.async_get(hass)
+    tplink_devices = dr.async_entries_for_config_entry(device_registry, entry.entry_id)
+    qty_tplink_devices = len(tplink_devices)
```

Comment:
`qty_` is not commonly used in homeassistant for signifying the number of elements, so I'd simply use `device_count` or something similar here.